src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -643,6 +646,7 @@ public FormValidation doTestConnection(@QueryParameter String name, @QueryParame
 
                 // test listing pods
                 client.pods().list();
+                client.close();
```

Comment:
Use the try-with-resources construct instead. https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html